.gitlab-ci.yml

```diff
@@ -98,6 +99,8 @@ test:acceptance:
       --mender-connect-deb-version $(cat ${CI_PROJECT_DIR}/output/mender-connect-deb-version)
       --mender-configure-version $MENDER_CONFIGURE_VERSION
       --mender-configure-deb-version $(cat ${CI_PROJECT_DIR}/output/mender-configure-deb-version)
+      --mender-monitor-version $MENDER_MONITOR_VERSION
+      --mender-monitor-deb-version $(cat ${CI_PROJECT_DIR}/output/mender-monitor-deb-version)
```

Comment:
(you probably figured this out yourself, but you need to add new these new arguments to `conftest.py`)